src/main/resources/mainnet.conf

```diff
@@ -25,6 +25,22 @@ ergo {
       # Difficulty for Autolykos version 2 activation (corresponding to ~ 1 TH/s hashrate)
       version2ActivationDifficultyHex = "6f98d5000000"
     }
+
+    reemission {
+      checkReemissionRules = false
+
+      emissionNftId = "20fa2bf23962cdf51b07722d6237c0c7b8a44f78856c0f7ec308dc1ef1a92a51"
+
+      reemissionTokenId = "d9a2cc8a09abfaed87afacfbb7daee79a6b26f10c6613fc13d3f3953e5521d1a"
+
+      reemissionNftId = "d3feeffa87f2df63a7a15b4905e618ae3ce4c69a7975f171bd314d0b877927b8"
+
+      activationHeight = 777217
+
+      reemissionStartHeight = 2080800
+
+      injectionBoxBytesEncoded = "80a8d6b9071003040005808098f4e9b5ca6a0402d1ed91c1b2a4730000730193c5a7c5b2a47302008f9e2d0220fa2bf23962cdf51b07722d6237c0c7b8a44f78856c0f7ec308dc1ef1a92a5101d9a2cc8a09abfaed87afacfbb7daee79a6b26f10c6613fc13d3f3953e5521d1a808088fccdbcc32300fca71b8b95f6ad14ce600a126c8842334d40d35f8754176c4cda2c95219f19f700"
```

Comment:
Does this bytes require update after ErgoTrees of the contracts became v1? (Note, the propertyBytes of ErgoTree does include the version)

Reply:
Injection box already in the chain, its script is v0. Will be used once only